satchless/payment/__init__.py

```diff
@@ -8,7 +8,7 @@ class PaymentsGatewayReceiptFormError(PaymentFailure):
     def __init__(self, form):
         error_message = u', '.join(
             [u':'.join([key, u', '.join(errors)])
-             for key, errors in form._errors.iteritems()])
+             for key, errors in form.errors.iteritems()])
```

Comment:
There's a subtle difference in behavior here: `errors` returns everything from `_errors` except the `__all__` key; to maintain the old behavior, you'd need to merge `errors` and `non_form_errors`, placing `non_form_errors` under the `__all__` key.